packages/shared/src/components/ExtensionOnboarding.tsx

```diff
@@ -37,7 +37,7 @@ const ExtensionOnboarding = (): ReactElement => {
         The magic awaits inside! ✨
       </p>
 
-      <Link href={onboardingUrl} passHref>
+      <Link href={`${onboardingUrl}?r=extension`} passHref>
```

Comment:
we could check if extension is installed by trying to post a message to it and then extension can reply for our domain with something so that we know about it

Reply:
if we are going with query params route I would recommend constructing the URL with `new URL()` since `onboardingUrl` could have query params already and then `?r` would break the syntax